composer.json

```diff
@@ -38,7 +38,8 @@
         "vimeo/psalm": "^4.6",
         "friendsofphp/php-cs-fixer": "^2.16",
         "phpunit/phpunit": "^7.5|^9.0",
-        "mikey179/vfsstream": "^1.6"
+        "mikey179/vfsstream": "^1.6",
+        "phpspec/prophecy": "^1.10"
```

Comment:
why?

Reply:
I suppose this error was the reason 
https://github.com/phparkitect/arkitect/runs/8292607729?check_suite_focus=true